frontend/playwright-globalSetup.ts

```diff
@@ -25,11 +26,12 @@ const command = new GetSecretValueCommand(secretValueRequest);
 
 module.exports = async () => {
   try {
-    const secret = JSON.parse((await client.send(command)).SecretString);
+    const secret = JSON.parse(
+      (await client.send(command)).SecretString || "null"
+    );
     process.env.TEST_ACCOUNT_USER = secret.test_account_username;
     process.env.TEST_ACCOUNT_PASS = secret.test_account_password;
   } catch (error) {
     console.error(error);
   }
-  process.env.STORAGE = JSON.stringify(featureFlags);
```

Comment:
we no longer need to store `featureFlags` as a env variable, since `frontend/playwright.config.ts` can import the file directly